test-api-generator/citrus-test-api-generator-maven-plugin/pom.xml

```
@@ -0,0 +1,124 @@
+<project
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+>
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>citrus-test-api-generator</artifactId>
+        <groupId>org.citrusframework</groupId>
+        <version>4.3.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>citrus-test-api-generator-maven-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
+
+    <name>Citrus :: Test API Generator :: Maven Plugin</name>
+    <description>Maven Plugin for generation of Citrus Test API</description>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-plugin-plugin</artifactId>
+                <version>3.10.2</version>
+                <configuration>
+                    <skipErrorNoDescriptorsFound>true</skipErrorNoDescriptorsFound>
+                </configuration>
+            </plugin>
+
+        </plugins>
+    </build>
+
+    <properties>
+        <swagger.version>2.2.21</swagger.version>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.citrusframework</groupId>
+            <artifactId>citrus-test-api-generator-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.swagger.core.v3</groupId>
+            <artifactId>swagger-core</artifactId>
+            <version>${swagger.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.swagger.core.v3</groupId>
+            <artifactId>swagger-models-jakarta</artifactId>
+            <version>${swagger.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.16.1</version>
```

https://github.com/citrusframework/citrus/blob/e05ec958626719c3f3eca8eecc2bac70f2ecf4b6/pom.xml#L203:

```suggestion
            <version>${commons.io.version}</version>
```

---

I replaced all explicit versions with properties, which are defined on root pom. Added the missing ones.